File: hw/medina/U1/hw09/farmSimulator.py
from datetime import datetime

class Chicken:
    def __init__(self, id, name, color, age, molting, born_on_date):
        self.id = id
        self.name = name
        self.color = color
        self.age = age
        self.molting = molting
        self.born_on_date = born_on_date

    def __str__(self):
        return f"Chicken{{id={self.id}, name='{self.name}', color='{self.color}', age={self.age}, molting={self.molting}, bornOnDate={self.born_on_date}}}"

if __name__ == "__main__":
    print("Simone's Chicken Farm Simulator V 0.5")
    chicken = Chicken(1, "Lucy", "white and brown", 0, True, datetime.now())
    
    print("chicken 1--> \n", chicken)

    id = 2
    name = "Maruja"
    color = "black"
    age = 1
    molting = False
    born_on_date = datetime.now()

    chicken = Chicken(id, name, color, age, molting, born_on_date)
    print("chicken 2--> \n", chicken)
    chicken1 = Chicken(1, "Josefina", "brown", 3, False, datetime.now())
    print("chicken 3--> \n", chicken1)

File: ws/medina/U1/ws9/ChickenFarmSimulator_/src/ec/edu/espe/simulator/view/FarmSimulator.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package ec.edu.espe.simulator.view;

import static com.sun.org.apache.xalan.internal.lib.ExsltSets.difference;
import ec.edu.espe.farmsimulator.model.Chicken;
import static java.lang.Double.sum;
import java.util.Date;

/**
 *
 * @author Simone Medina, JavaSquad, DCCO-ESPE
 */
public class FarmSimulator{ 
        public static void main (String[] args){
            System.out.println("Simone's Chicken Farm Simulator V 0.5");
            Chicken chicken= new Chicken(1,"Lucy", "white and brown",0, true, new Date());
            Chicken chicken1= new Chicken(3,"Josefina", "brown",3, false, new Date());
            System.out.println("chicken--> \n"+ chicken);
            
            int id=2;
            String name="Maruja";
            String color="black";
            int age=1;
            boolean molting=false;
            Date bornOnDate= new Date();
            
            
            chicken = new Chicken(id, name, color, age, molting,bornOnDate);
            System.out.println("chicken 2--> \n"+ chicken);
            System.out.println("chicken 2--> \n"+ chicken1);
    
    
}
Short minuend; 
    short subtrahend;
    short difference;
    

}
